glue/core/data_factories/gridded.py

```diff
@@ -71,10 +71,8 @@ def is_gridded_data(filename, **kwargs):
 
 gridded_data.label = "FITS/HDF5 Image"
 gridded_data.identifier = is_gridded_data
+gridded_data.priority = 2
```

Comment:
Ah I see, we are not always adding 'item' namedtuples to the factory registry, so `priority` may not be defined. Is that the issue?